modules/@apostrophecms/import-export-asset/index.js

```diff
@@ -0,0 +1,6 @@
+module.exports = {
+  improve: '@apostrophecms/asset',
+  icons: {
```

Comment:
You can skip the improve of asset by adding this in `index.js` where the icon is used

Reply:
Removed and added in the main index.js